component.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.sidebar {
+  margin: 0;
```

Comment:
You can include this in reset.css and import that file in this component. So that you won't have to include `margin: 0` `padding:0` everywhere.